draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -556,6 +556,12 @@ Catalog zones do not need to be signed using DNSSEC, their zone transfers being
 authenticated by TSIG.  Signed zones MUST be handled normally by nameservers,
 and their contents MUST NOT be DNSSEC-validated.
```

Comment:
I suggest to remove the last sentence. (It is unnecessary if it is operationally agreed that catalog zones aren't signed, so nobody validates them. But I don't get the rationale for an eternal prohibition.)

Reply:
/me agrees!